docs/configuration-reference/components/velero.md

```diff
@@ -31,64 +31,64 @@ Velero component configuration example:
 # velero.lokocfg
 component "velero" {
 
-  # provider = "azure/openebs/restic"
```

Comment:
Nit: I suggest opening separate PRs with such fixes, so they can be immediately merged.